setup.cfg

```diff
@@ -9,7 +9,13 @@ max-line-length = 120
 
 [flake8]
 max-line-length = 120
-ignore = F401,E402,F403,W503,W504,F821
+ignore = E203, E402, W503, W504, F821
+per-file-ignores =
+    __init__.py: F401, F403, F405
+    ./hubconf.py: F401
+    torchvision/models/mobilenet.py: F401, F403
+    torchvision/models/quantization/mobilenet.py: F401, F403
+    test/smoke_test.py: F401
```

Comment:
See https://github.com/pytorch/vision/pull/4178#discussion_r669383913 for details on this change.